semantic-core/src/Analysis/Concrete.hs

```diff
@@ -21,45 +21,45 @@ import           Control.Effect.Reader hiding (Local)
 import           Control.Effect.State
 import           Control.Monad ((<=<), guard)
 import           Core.File
-import           Core.Name
 import           Data.Function (fix)
 import qualified Data.IntMap as IntMap
 import qualified Data.IntSet as IntSet
 import qualified Data.Map as Map
 import           Data.Semigroup (Last (..))
 import qualified Data.Set as Set
+import           Data.String (IsString)
```

Comment:
We need this because we currently hard-code some stuff relating to record lookups and inheritance, and thus need to be able to construct a concrete name from a `String`. This _should_ be an implementation detail of the evaluator &/or compilation, so we should hopefully be able to ditch that requirement sooner or later.

Reply:
Can we file an issue about this so it doesn’t get lost in the shuffle?